tsc/README.md

```diff
@@ -0,0 +1,27 @@
+# Technical Steering Committee
+
+The KubeEdge Technical Steering Committee (TSC) is the governing body for the KubeEdge project, providing decision-making and oversight pertaining to the KubeEdge project bylaws. The Technical Steering Committee also defines the project values and structure.
+
+ See the [Technical Steering Committee Charter](./charter.md) for specific committee structure information.
+
+## Members
+
+The initial TSC members are picked by the [project maintainers](https://github.com/kubeedge/kubeedge/blob/v1.11.0/MAINTAINERS.md) based on contributions and experience.
+
+**Initial TSC members**
+- Benjamin Huo - KubeShpere
+- Fei Xu - Huawei
+- Hongbing Zhang - DaoCloud
+- Huan Wei - HarmoyCloud
+- Kevin Wang - Huawei
+- Tina Tsou - Arm
+- Yin Ding - Google
+
+
+*We would like to have 2-year terms finally, and for the first year, we can have half seats 2-year term and half seats 1-year term.*
+
+**The first election will come in Sep/Oct 2024.**
```

Comment:
`for the first year, we can have half seats 2-year term and half seats 1-year term.*`
The above statement means the first election will come in 2023?

Reply:
Updated the line above with: *We would like to have 2-year terms finally, and for the first election, we can have 4 seats 2-year term and 3 seats 1-year term.*